rules/carbonblack_rules/cb_audit_admin_grant.py

```diff
@@ -0,0 +1,25 @@
+PREFIXES = ("Updated grant: ", "Created grant: ")
+
+
+def rule(event):
```

Comment:
Something like this may consolidate things a bit:
```py
def rule(event):
    desc = event.get("description", "")
    if "Admin" in desc:
        return True
    if not any(
        event.get("requestUrl", "").startswith("/access/"),
        any(desc.startswith(prefix) for prefix in PREFIXES),
    ):
        return False
    return False
```

Reply:
Putting the `return True` first changes the logic a bit, and could cause false positives if the admin role is deleted for example.  I do like the `any` nesting and will try to consolidate while preserving the logic.

Reply:
Makes sense -- I wasn't considering the implications of moving that up higher.